DUI3-DX/DUI3/Speckle.Connectors.DUI/Bridge/BrowserBridge.cs

```diff
@@ -288,7 +291,7 @@ public void Send(string eventName)
   {
     var script = $"{FrontendBoundName}.emit('{eventName}')";
 
-    _scriptMethod!(script);
+    _scriptMethod?.Invoke(script);
```

Comment:
_scriptMethod should have been initialized at this point. If not we should throw